opentelemetry-api/src/opentelemetry/context/__init__.py

```diff
@@ -11,25 +11,22 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
-
 """
 The OpenTelemetry context module provides abstraction layer on top of
 thread-local storage and contextvars. The long term direction is to switch to
 contextvars provided by the Python runtime library.
 
 A global object ``Context`` is provided to access all the context related
-functionalities::
+functionalities:
```

Comment:
Did you look at the rendered Sphinx documentation? These double-colons are not typos but rst-Syntax for code blocks.

Reply:
Thanks! I am aware they are rst-syntax. I don't believe I modified this by hand: most likely an autoformatter did it since I touched this file. I'll take a look.